shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -493,43 +493,19 @@ - (BOOL)createShell:(NSString*)entrypoint libraryURI:(NSString*)libraryURI {
                                                      shell.GetIsGpuDisabledSyncSwitch());
       };
 
-  if (flutter::IsIosEmbeddedViewsPreviewEnabled()) {
```

Comment:
wow. 😃 does it mean we support dynamic thread merging on iOS after this is in?

Reply:
Yeah this is the last puzzle we needed to solve to support dynamic thread merging.